app/main.js

```diff
@@ -500,3 +544,10 @@ server.listen(port, () => {
 /* Utility */
 
 var stripIPv6 = ip => ip.replace(/^.*:/, '')
+
+/* Neccessary Function to fetch rotation */
+function getRotationFromDeviceID(id) {
+  for (var i = 0; i <= devices.length; i++) {
+
```

Comment:
This function appears to be an artifact from an alternative approach? Doesn't look like its ever called.

Additionally, I like the cleanup changes you've made, but I prefer to use ES6 arrow notation for functions, and save them to variables instead of declare them as functions (see `stripIPv6` directly above).